### DISCUSSION TITLE
FileUpload Filament xlsx

### DISCUSSION THREAD

**Discussion by nimahamidpour1994** [filamentphp/filament#10524] [Help]
when i want to upload excel xlsx with FileUpload in handleRecordCreation get 
Could not find zip member zip:///var/www/html/storage/framework/cache/laravel-excel/laravel-excel-AD5wvtZGEgM8jw1ZXhVtB0fxkO8O0CFs.xlsx#_rels\.rels

**Comment by leandrocfe** [comment]
Is there the php-zip extension installed?
